infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LayoutSearchController.kt

```diff
@@ -0,0 +1,47 @@
+package fi.fta.geoviite.infra.tracklayout
```

Comment:
Mielipiteitä tällaisesta tiedostosta/nimityksestä? Pitäisikö tämän tiedoston sisältämän yksittäisen endpointin sijaita kuitenkin jossain toisessa tiedostossa?

Reply:
Vaikka yleisesti ottaen yksittäisen funktion/endpointin tiedostot onkin vähän ärsyttäviä, niin mielestäni tää on ihan jees. Tää kuitenkin grouppaa selkeästi yhteen sellaista kamaa mille ei ole aiemmin ollut omaa controlleria/pathia, kun kaikille eri asset-tyypeille on omat controllerinsa.

Yksi vaihtoehto olisi tämän renameisi uudestaan esim. ihan vaan `SearchController`:iksi ja alkaisi yhdistelemään meidän muitakin hakuja tämän alle. En tosin ole satavarma onko se järkevää, mutta tuollainen tuli ekana mieleen kun tätä luin.

Reply:
Minustakin näyttää ihan hyvältä näin. Jos tänne jotain haluaisi yhdistää niin ehkä nuo layoutin yksittäiset searchit voisi ottaa tänne käsitteiden omilta. Mutta sekin on vähän kumsi-kumsa. En lähtisi layoutin ulkopuolisia juttuja enää tänne vetämään koska ne ovat niin eri kokonaisuutta jo. Eli hyvä näin minusta.